wifiphisher/common/accesspoint.py

```diff
@@ -151,3 +153,37 @@ def on_exit(self):
         # sleep 2 seconds to wait all the hostapd process is
         # killed
         time.sleep(2)
+
+    #starts background thread that reads dnsmasq file with clients connected
+    def init_thread_read_victims_file(self):
+        thread = threading.Thread(target=self.read_connected_victims_file, args=())
```

Comment:
@sophron @blackHatMonkey in order to complete this feature, here we need to create another thread to read the dnsmasq file. This will complicate our program and reduce the efficiency. I still think currently we have `Roguehostapdinfo` extension module, and we will maintain a `MAC to ssid dictionary` and also prints some victims information. this feature may be better implemented there, for one thing is we don't need to create another thread and  for another we make the code of printing info more concentrated. But any way, besides that this PR looks fine for me :)

Reply:
Yes, that's a valid point.

Is it the <a href="https://github.com/wifiphisher/roguehostapd/blob/master/roguehostapd/apctrl.py#L82">roguehostapd.get_karma_data()</a> the right method to use here?

Reply:
Thinking of it again, if we will use `roguehostapd` here, we are essentially throwing away the `force-hostapd` option. But I guess, now that `roguehostapd` has become essential part of the tool, we can remove the `force-hostapd` option. What do you think?